debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -43,29 +44,32 @@
 /**
  * MySQL-specific customization of the conversions from JDBC values obtained from the MySQL binlog client library.
  * <p>
- * This class always uses UTC for the default time zone when converting values without timezone information to values that require
+ * This class always uses UTC for the default time zone when converting values without timezone information to values
+ * that require
```

Comment:
There are too many unnecessary formatting changes in this diff, and it's impossible to easily see what changes are important for this fix. Can you revert most of these?